kalite/distributed/templates/distributed/homepage.html

```diff
@@ -62,14 +62,14 @@ <h2 class="playlist-list-header"><%= title %></h2>
     <div class="container">
 
         <div class="row">
-            <h2 class="main-headline hidden-xs">{{ channel_data.head_line }}</h2>
```

Comment:
I don't think the h1 tag is appropriate here, the default content for this is "A free world-class education for anyone anywhere." which is not a page heading (h1) but rather a subheading - <p> may be more appropriate.